website/scripts/sync-algolia-crawler.mjs

```
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
```

ESLint

[error] This file needs no shebang.

(n/hashbang)